CONTRIBUTING.md

```diff
@@ -454,16 +498,20 @@ The following parameters are available:
 
 ```md
 <!-- Provide keybinding for one OS and another for all others -->
+
 {{< keybind mac="⇧⌘P" other="Ctrl+Shift+P" >}}
 
 <!-- Provide a keybind for all OSs -->
+
 {{< keybind all="Ctrl+Shift+P" >}}
 
 <!-- Provide unique keybindings for each OS -->
+
 {{< keybind mac="⇧⌘P" linux="Ctrl+Shift+P" win="Ctrl+Shift+Alt+P" >}}
 ```
```

Comment:
I'm worried that this will add newlines after all comments in code blocks. This is only a stylisic preference, but I feel like this will add a lot of unnecessary vertical height to existing code examples.